en/docs/learn/configuring-sms-otp.md

```diff
@@ -1,210 +1,160 @@
-			</tr>
-			<tr>
-				<td>SMSOTPMandatory</td>
-				<td>If the value is true, the second step will be enabled by the admin. The user cannot be authenticated without SMS OTP authentication. This parameter is used for both super tenant and tenant in the configuration. The value can be <code>true</code> or <code>false</code>.</td>
-			</tr>
-			<tr>
-				<td>CaptureAndUpdateMobileNumber</td>
-				<td>When <code>SMSOTPMandatory</code> is set to <code>true</code> and the user forgets to update the mobile number in a specific user profile where this property is set to <code>true</code>, the user can update a mobile claim with value in authentication time (If it is first login) and get the mobile number from the user's profile to send OTP. This update functionality will happen in the first login only. Once the user updates the mobile number, in the next login, the mobile number will take the updated mobile number from that specific user profile.</td>
-			</tr>
-			<tr>
-				<td>SendOTPDirectlyToMobile</td>
-				<td>When <code>SMSOTPMandatory</code> is set to <code>true</code> and the user does not exist in the user store and if the admin sets <code>SendOTPDirectlyToMobile</code> to <code>true</code>, the user can enter the mobile number in authentication time in a mobile number request page; the OTP will be directly sent to that mobile number.</td>
-			</tr>
-			<tr>
-				<td>redirectToMultiOptionPageOnFailure</td>
-				<td></td>
-		</tbody>
-	</table>
-
-	!!! note
-
-		The SMS provider is the entity that is used to send the SMS. The SMSOTP connector has been configured such that it can be used with most types of SMS APIs. Some use the GET method with the client secret and API Key encoded in the URL (e.g., Nexmo), while some may use the POST method when sending the values in the headers and the message and telephone number in the payload (e.g., Clickatell). Note that this could change significantly between different SMS providers. The configuration of the connector in the identity provider would also change based on this.
+    -   To ensure you get the full understanding of configuring SMS OTP with WSO2 Identity Server, 
+        the sample travelocity application is used in this use case. Therefore, make sure to 
+        [download the travelocity 
+        application](https://maven.wso2.org/nexus/content/groups/public/org/wso2/is/org.wso2.sample.is.sso.agent/5.9.0/).
+    -   The samples run on the Apache Tomcat server and are written based on Servlet 3.0. Therefore, 
+        download Tomcat 8.x from [here](https://tomcat.apache.org/download-80.cgi) .
+    -   Install Apache Maven to build the samples. For more information, 
+        see [Installation Prerequisites](../../setup/installation-prerequisites).
```

Comment:
Do we need to install maven, since we don't need to build the sample now?

Reply:
Fixed: 8ae37d5a108f6a539b0db845f5b2f5704be01bd1